app/views/kondos/index.html.erb

```diff
@@ -1,15 +1,16 @@
 <div class="container">
-  <div class="row justify-content-center">
-    <div class="col-6">
-      <% @kondos.all.each do |kondo| %>
-      <div>
-        <h4><%= link_to kondo.name, kondo_path(kondo) %></h4>
-        <h5><%= kondo.summary %></h5>
-        <span><%= kondo.prefecture %></span>
-        <span><%= kondo.user.first_name %></span>
-        <span><%= kondo.price %></span>
-      </div>
-      <% end %>
-    </div>
+  <div class="row">
+    <% @kondos.each do |kondo| %>
+      <div class="col-xl-3 col-md-6 col-sm-12">
```

Comment:
@hiroya210 I like how the cards look man!

Played with this locally and I noticed that the centering becomes off when resizing the window. That said what I could recommend is to add the following classes on this div.

```html
<div class="col-xl-3 col-md-6 col-sm-12 d-flex justify-content-center">
```
or anywhere else that you'd find appropriate as long as the centering would be fine after resize

Reply:
Thanks, its fixed now.